src/scale-manager/ScaleManager.js

```diff
@@ -29,11 +29,13 @@ export class ScaleManager {
     const width = event.target.innerWidth;
     const height = event.target.innerHeight;
 
-    this.callback({
-      width,
-      height,
-      ratio: width / height
-    });
+    setTimeout(() => {
```

Comment:
Would it be worth making this conditional depending on the browser?

Reply:
True, SpringRoll 1 did this as well (do the timeout for all browsers) and I'm fine with that.